core/src/test/scala/unit/kafka/common/TopicTest.scala

```diff
@@ -42,7 +43,7 @@ class TopicTest {
         fail("Should throw InvalidTopicException.")
       }
       catch {
-        case e: InvalidTopicException => "This is good."
+        case e: errors.InvalidTopicException => "This is good."
```

Comment:
An existing issue, but would you mind replacing "This is good" by a comment? Code like that can cause compiler warnings as the string is never used (it's also weird).